test/test_aug_caching.py

```diff
@@ -0,0 +1,67 @@
+import os
+
+import numpy as np
+from torch.utils.data import Dataset
```

Comment:
Do you need any functionality of this parent class? Could this be omitted?

Reply:
if you mean Dataset: it was not necessary and I removed it. Thanks!